src/Plugin/TwigPlugin/JumpstartUITwig.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\jumpstart_ui\Plugin\TwigPlugin;
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Extend Drupal's Twig_Extension class.
+ */
+class JumpstartUITwig extends \Twig_Extension {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFunctions() {
+    return [
+      new \Twig_SimpleFunction('getUUID', [$this, 'getUuid']),
```

Comment:
to me `getUUID`  is misleading with what the method does. `getUniqueId` would be more appropriate to me.

Reply:
in fact this raises some concern to me. How does this play out among different pages? after I read [this stackoverflow](https://drupal.stackexchange.com/questions/281272/how-can-i-use-htmlgetuniqueid-in-twig) i'm concerned with this. I tested the theory and putting this on some areas produces duplicate IDs. an easy way to test is to add `{% set attributes = attributes.setAttribute('id', getUUID('mikes-block')) %}` to the block.html.twig. Add a block or two to various pages and inspect the IDs. I've found multiple `id="mikes-block"` elements

Reply:
Aye, we came across this in the Earth project as well. This function can also produce duplicates if used incorrectly in hooks or other places where sessions change or caches get cleared. This is one of those tools where you have to use cautiously and understand that you don't have the guarantee it will work. How about we name it to `getMaybeAUniqueId`?

Reply:
:laughing: I think its just important to ensure we don't use it often.